app/src/main/java/org/stacks/app/shared/Exceptions.kt

```diff
@@ -0,0 +1,4 @@
+package org.stacks.app.shared
+
+//Exceptions
+class IdentitiesParsingFailed(e: Exception) : Exception(e)
```

Comment:
Feels a bit odd having a file for all the Extensions on the app, for all the different layers. But no big deal.